spec/fixtures/stubs/unlinked_defendant.json

```diff
@@ -40,8 +40,12 @@
         "order_index": 1,
         "title": "Copying false instrument with intent",
         "mode_of_trial": "Indictable only",
-        "mode_of_trial_reason": "Court directs trial by jury",
-        "mode_of_trial_reason_code": "05",
+        "mode_of_trial_reasons": [
+          {
+            "code": "gfc0",
```

Comment:
use a code that is of the format expected such as `05` - as i believe it will be that. there is a spreadsheet around somewhere

Reply:
👍  am using '4' (from the spreadsheet)